thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/RCAFrameworkRunner.java

```diff
@@ -139,7 +144,12 @@ public static void main(String[] args) throws Exception {
 
     ThirdEyeConfiguration thirdEyeConfig = new ThirdEyeAnomalyConfiguration();
     thirdEyeConfig.setRootDir(config.getAbsolutePath());
-    ThirdEyeCacheRegistry.initializeCaches(thirdEyeConfig);
+    String clientConfigPath = thirdEyeConfig.getClientsPath();
+    ClientConfig clientConfig = ClientConfigLoader.fromClientConfigPath(clientConfigPath);
+    if (clientConfig == null) {
+      throw new IllegalStateException("Could not create client config from path " + clientConfigPath);
+    }
+    ThirdEyeCacheRegistry.initializeCaches(thirdEyeConfig, clientConfig);
```

Comment:
This kind of code repeats over and over. I wonder if initializeCaches() should simply extract this information from thirdEyeConfig by itself.